### ISSUE TITLE
Font Not Saving

### ISSUE THREAD

**Issue by gutohertzog** [terrychou/iVim#272]
I downloaded a Nerd Font and changed in the Font Settings (regular, bold and italic) and/or the code with

```VimScript
set guifont=cascaydiacove_nerd_font:h13
```

But every time I quit the app with `:wqa` or `swiping up`, all the fonts come back to Source Code Pro. 

If I set up manually bold and italic, they keep the Nerd Font, but regular keep coming back to Source Code Pro with size 14.

The print after the restart: 
![image](https://github.com/terrychou/iVim/assets/7025057/f8666c9d-3617-43b9-a57f-b51de781cffc)

**Comment by wturrell** [comment]
Yes, I have this problem as well (version 2.55 on latest iPadOS 17.6.1).
Except this is with the default font and the font size changes are still lost regularly.
